client/scss/components/_page_description.scss

```diff
@@ -31,7 +35,7 @@
 }
 
 .page-description__row {
-  margin-bottom: 0.6em;
+  margin-bottom: 0.3em;
```

Comment:
Okay. Just seems weird that we seem to be using a similar hack all over the show, which means we could probably formalise it as `line-height` etc are all measurements that we should be able to conceptualise into a pattern.

I say "probably", and "should", as I have no idea how though.